src/backend/app.js

```diff
@@ -21,10 +22,85 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//Respond with all meals in the future (relative to the when datetime)
+app.get("/future-meals", async (req, res) => {
+  try {
+    const today = new Date();
+    const futureMeals = await knex("Meal").where("_when", ">", today).select();
+    if (futureMeals.length === 0) {
+      res
+        .status(404)
+        .json({ data: null, message: "No future meal available." });
+      return;
+    }
```

Comment:
The assignment states that in case of queries where multiple meals should be returned, and there are no meals, the server should return just an empty array. You chose to instead return a status 404, and an appropriate message. This makes sense as well, but the empty array is a valid answer, and lets the user know that there are no meals. 😃